dh_collections_prism_collection_community.json

```diff
@@ -81,7 +81,7 @@
     "community_id": "researchers-collections"
   },
   "3ca02e5e-83be-4ea7-b51f-11aee3497e6c": {
-    "collection_id": "Christian Adames Publications & Presentations",
+    "collection_id": "Christian Adames Publications and Presentations",
```

Comment:
:+1: great to catch it early (the import doesn't do anything with the collection id yet, but it would probably also succumb to that issue if it was around)